postgres-appliance/Dockerfile

```diff
@@ -513,9 +515,19 @@ WORKDIR $PGHOME
 COPY motd /etc/
 COPY runit /etc/service/
 COPY pgq_ticker.ini $PGHOME/
+COPY scripts bootstrap /scripts/
 
 RUN sed -i "s|/var/lib/postgresql.*|$PGHOME:/bin/bash|" /etc/passwd \
-        && chown -R postgres:postgres $PGHOME $RW_DIR \
+        && if [ "$COMPRESS" != "true" ]; then \
+           # non-root prep
+           mkdir -p $PGHOME \
+           && sed -i "s|/var/lib/postgresql.*|$PGHOME:/bin/bash|" /etc/passwd \
+           # Set permissions
+           && chmod 775 $PGHOME \
```

Comment:
PGHOME already exists after https://github.com/zalando/spilo/pull/414/files#diff-82f985fcf1262fb2d3d1843b9e74d9c3R517 and its permissions already 755

The `/etc/passwd` already contains the correct home directory for postgres: https://github.com/zalando/spilo/pull/414/files#diff-82f985fcf1262fb2d3d1843b9e74d9c3R520

Reply:
You're right, I'll remove this (just tested it okay).